superset-frontend/src/components/TableSelector.jsx

```diff
@@ -274,12 +274,15 @@ export default class TableSelector extends React.PureComponent {
   }
 
   renderDatabaseSelect() {
+    const filters = this.props.formMode
+      ? ''
+      : 'filters:!((col:expose_in_sqllab,opr:eq,value:!t)),';
```

Comment:
Not sure what we've been doing so far around building API GET urls, but generally I would rather see `rison.encode` used against JS data structures instead of raw `rison` as many engs may not be familiar with `rison`